### ISSUE TITLE
add more metric plugins

### ISSUE THREAD

**Issue by massachusett** [broadinstitute/DeepProfiler#107]
http://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics

**Comment by massachusett** [comment]
metrics.auc
metrics.average_precision_score
metrics.brier_score_loss
metrics.cohen_kappa_score
metrics.confusion_matrix
metrics.f1_score
metrics.fbeta_score
metrics.hinge_loss
metrics.precision_recall_curve
metrics.roc_auc_score
metrics.roc_curve
